precompiles/xvm/evm_sdk/flipper.sol

```diff
@@ -2,17 +2,18 @@
 
 interface XVM {
     function xvm_call(
-        uint8 calldata vm_id,
+        uint8 vm_id,
         bytes calldata to,
         bytes calldata input,
-    ) external;
+        uint256 value
+    ) external payable returns (bool success, bytes memory data);
 }
 
 library Flipper {
     const XVM XVM_PRECOMPILE = XVM(0x0000000000000000000000000000000000005005);
 
     function flip(bytes to) {
         bytes input = "0xcafecafe";
-        XVM_PRECOMPILE.xvm_call(0x1F, to, input);
+        XVM_PRECOMPILE.xvm_call(0x1F, to, input, 1000000);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `flip` function in the `Flipper` library has been updated to include the `value` parameter when calling the `xvm_call` function. The value `1000000` is passed as the `value` argument. Please verify that this change aligns with the intended behavior and requirements of the `flip` function.

```diff
     function flip(bytes to) {
         bytes input = "0xcafecafe";
        XVM_PRECOMPILE.xvm_call(0x1F, to, input, 1000000);
     }
```

LGTM!


<!-- This is an auto-generated comment by OSS CodeRabbit -->